### PR TITLE
Fix #5325 : added --keep flag to dvc exp remove documentation

### LINKED ISSUE #5325: Update documentation for fix #10593
## Description

Issue iterative/dvc#10593 addresses the need for a --keep flag on `dvc experiments remove` to keep the last `n` experiments instead of deleting them.
A PR has been submitted to do so. This issue goes hand in hand with it to update the corresponding documentation.

### Possible documentation update

--keep : use this flag keep the experiments selected using other options (--rev, --num, or experiment names) and remove all others.

### PR DESCRIPTION
**Pull request by rmic**
## Description 

Fix #5325 : added --keep flag to dvc exp remove documentation as needed by the corresponding PR in the code : https://github.com/iterative/dvc/pull/10633

### CODE REVIEW

File: content/docs/command-reference/exp/remove.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ With `--queue`, the list of experiments awaiting execution is cleared instead.
 - `-g`, `--git-remote` - Name or URL of the Git remote to remove the experiment
   from
 
+- `--keep` - keep only the selected experiments and remove all others
```
Comment (shcheklein) [inline content/docs/command-reference/exp/remove.md:43]: minor / stylistic:

how about this:

> changes the default behavior to the opposite for options like `-n`, `--rev`. For example `-n <num> --keep` _keeps_ experiments from the last `num` commits and _removes all_ other experiments.


does it change behavior for `--queue`? what is the behavior for `-A`?

Comment (rmic) [reply content/docs/command-reference/exp/remove.md:43]: As of now, it raises an exception with `--queue` (InvalidArgumentError), until we decided what to do with `--queue`  as mentioned in [dvc - #10639](https://github.com/iterative/dvc/issues/10639)

As one might expect, `--keep -A` (or `--keep --all-commits`) keeps all experiments and removes nothing, basically just wasting a bit of energy, iops and cpu cycle. In terms of experiments and data, this has the same outcome as not running the command at all.

File: content/docs/command-reference/exp/remove.md [hunk 2]
```diff
@@ -125,3 +125,24 @@ $ dvc exp list myremote
 master:
         conic-ease
 ```
+
+Conversely, to keep only specific experiments (and remove all others), give their names to `dvc exp remove` with the `--keep` flag :
```
Comment (shcheklein) [inline content/docs/command-reference/exp/remove.md:129]: You would need to run linters most likey

File: content/docs/command-reference/exp/remove.md [hunk 3]
```diff
@@ -40,6 +40,10 @@ With `--queue`, the list of experiments awaiting execution is cleared instead.
 - `-g`, `--git-remote` - Name or URL of the Git remote to remove the experiment
   from
 
+- `--keep` - changes the default behavior to the opposite for options like -n, --rev.
```
Comment (shcheklein) [inline content/docs/command-reference/exp/remove.md:43]: -n -> `-n`
--rev -> `--rev`

(etc)

File: content/docs/command-reference/exp/remove.md (hunk 2 above)
Comment (shcheklein) [reply content/docs/command-reference/exp/remove.md:129]: yep, thanks probably this should have some instruction on how to setup the project https://dvc.org/doc/contributing/docs

Comment (rmic) [reply content/docs/command-reference/exp/remove.md]: Thanks ! That helped a lot !

**Review by shcheklein — APPROVED** [review]

**Pull request merged**